lib/ca_testing/patches/selenium_manager.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+module CaTesting
+  module Patches
+    class SeleniumManager < Base
+      private
+
+      def description
+        <<~DESCRIPTION
+          This patch fixes an issue with Selenium4 not bubbling up the httpOnly property of a cookie
+          See: https://github.com/SeleniumHQ/selenium/pull/8958 for more details/discussion including this fix
+        DESCRIPTION
+      end
+
+      def perform
+        ::Selenium::WebDriver::Manager.prepend CookieConverter
+      end
+
+      def deprecate?
+        false
```

Comment:
Can we set the deprecated/prevented times to have the same date/times as the other selenium fix.

Reply:
Thanks, I wasn't sure about the deprecation date on this one. I'm going to change it to be the same as the other selenium fix.